File: Data Structures & Algorithms/online-stock-span/submission-0.cpp
class StockSpanner {
private:
    vector<int> v;
public:
    StockSpanner() {
        
    }
    
    int next(int price) {
        v.push_back(price);
        int cnt = 0;
        for(int i = v.size() - 1; i >= 0; i--) {
            if(price >= v[i]) ++cnt;
            else break;
        }
        return cnt;
    }
}; 

/**
 * Your StockSpanner object will be instantiated and called as such:
 * StockSpanner* obj = new StockSpanner();
 * int param_1 = obj->next(price);
 */

File: Data Structures & Algorithms/design-hashmap/submission-0.java
class MyHashMap {
    private List<List<Integer>> data;

    public MyHashMap() {
        data = new ArrayList<>();    
    }
    
    public void put(int key, int value) {
        for(List<Integer> pair : data) {
            if(pair.get(0) == key) {
                pair.set(1, value);
                return;
            }
        }
        List<Integer> pair = new ArrayList<>();
        pair.add(key);
        pair.add(value);
        data.add(pair);
    }
    
    public int get(int key) {
        for(List<Integer> pair : data) {
            if(pair.get(0) == key) {
                return pair.get(1);
            }
        }
        return -1;
    }
    
    public void remove(int key) {
       Iterator<List<Integer>> it = data.iterator();
       while(it.hasNext()) {
            List<Integer> pair = it.next();
            if(pair.get(0) == key) {
                it.remove();
            }
       } 
    }
}

/**
 * Your MyHashMap object will be instantiated and called as such:
 * MyHashMap obj = new MyHashMap();
 * obj.put(key,value);
 * int param_2 = obj.get(key);
 * obj.remove(key);
 */